# PulseAudio's build system doesn't propagate its include directories,
# so projects depending on PulseAudio won't be able to find PulseAudio's header files.
# This patch fixes that problem.

--- a/src/pulse/meson.build
+++ b/src/pulse/meson.build
@@ -94,7 +94,7 @@ libpulse = shared_library('pulse',
   vs_module_defs : 'libpulse.def',
 )
 
-libpulse_dep = declare_dependency(link_with: libpulse)
+libpulse_dep = declare_dependency(link_with: libpulse, include_directories: topinc)
 
 install_headers(
   libpulse_headers, 'simple.h',
